subsys/metrics/src/main/java/org/commonjava/indy/metrics/conf/IndyMetricsConfig.java

```diff
@@ -39,9 +39,164 @@
 
     public final static int INDY_METRICS_REPORTER_GRPHITEREPORTER_DEFAULT_PERIOD = 30;
 
+    public final static String INDY_METRICS_REPORTER_CONSOLEREPORTER = "console.reporter";
+
     public final static String INDY_METRICS_ISENABLED = "enabled";
+
     public final static String INDY_METRICS_REPORTER_ISENABLED = "reporter.enabled";
 
+    public final static String INDY_METRICS_REPORTER_ZABBIXREPORTER = "zabbix.reporter";
```

Comment:
These constant values will have to adjust to align with my comment above about the conf file.